runtime/power.S

```diff
@@ -367,7 +367,7 @@ FUNCTION caml_call_realloc_stack
    /* Save all registers, as well as ALLOC_PTR and TRAP_PTR */
         SAVE_ALL_REGS  /* TMP2 is preserved */
```

Comment:
This comment should be adjusted too, since we no longer rely upon `TMP2` to be unchanged.
```suggestion
        SAVE_ALL_REGS
```

Reply:
OK, done.